src/style.css

```
@@ -1 +1,60 @@
 /* Styles go here */
+body {
+  margin: 0;
+  font-family: Roboto, Arial, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+.header {
+  display: flex;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  justify-content: space-between;
+  padding: 10px 50px;
```

the header has not top and bottom padding. Just all elements placed in the middle
```suggestion
  padding: 0 50px;
```